lesson_10/handlers/handlers.go

```diff
@@ -0,0 +1,180 @@
+package handlers
+
```

Comment:
This file is very hard to read, many types of responses are mixed with methods. I'll recommend to move all separate types into separate files. But it is up to you, in final project I will consider code like this as mistake